uwblueprint/jump-math
# feat: create admin login page frontend

## Notion ticket link

[Admin Login Page](https://www.notion.so/uwblueprintexecs/47bf4727a2444bccad4a91952898b8f3?v=9658436bf9be409e90581631a7155f16&p=775b7d517b544ef1a6e5ba48a613c372&pm=s)



## Implementation description
* created the admin login page based on the design 


## Steps to test
1.navigate to localhost:3000/login, verify whether the implementation matches the figma 
https://www.figma.com/file/cRjJxm8087kkymfH4ieKau/Platform-Working-File?node-id=860%3A46371&t=nkvEUyCzWBvXdL2T-0



## What should reviewers focus on?
* how to export the image from figma??


## Checklist
